p2p/server.go

```diff
@@ -521,6 +525,18 @@ func (srv *Server) setupLocalNode() error {
 }
 
 func (srv *Server) setupDiscovery() error {
+	srv.discmix = enode.NewFairMix(fallbackInterval)
```

Comment:
`fallbackInterval` is 20 seconds. That was meant as a final failure to connect to the bootnodes. I'm not sure that's a correct number here, as any hanging source iterator will block discovery for 20 secs each time. I'd expect something like 1 second timeout. Am I missing something?

Reply:
Timeout lowered to 5s.